kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -28,13 +28,29 @@
       </div>
 
       <div class="create">
-        <user-create-modal></user-create-modal>
+        <icon-button @click="openCreateModal" class="add-user-button" text="Add New" :primary="false">
```

Comment:
"openCreateModal" is a kind of confusing-sounding name - could be nice to make the named more consistent with the CSS. For example:

* `openAddUserModal`
* `closeAddUserModal`
* `openEditUserModal`
* `closeEditUserModal`